File: python/simpsons.py
def namelist(names):
	simpsons = [name["name"] for name in names]

	if len(simpsons) == 0:
		return ""
	elif len(simpsons) == 1:
		return str((simpsons)[0])
	elif len(simpsons) == 2:
		return str((simpsons)[0]) + " & " + str((simpsons)[1])
	else:
		return ", ".join(simpsons[0:-2]) + ", " + str((simpsons)[-2]) + " & " + str((simpsons)[-1])

File: js/simpsons.js
function list(names){
	let simpsons = names.map(i => Object.values(i));
	let x = simpsons.length;
	switch(x) {
		case 0:
		return ""
		break;
		case 1:
		return simpsons[0][0]
		break
		case 2:
		return simpsons[0] + " & " + simpsons[1]
		break
		default:
		return simpsons.slice(0, x-2).join(", ") + ", " + simpsons[x-2] + " & " + simpsons[x-1];
		}
}